win32ss/gdi/ntgdi/freetype.c

```diff
@@ -2272,6 +2272,10 @@ FillTM(TEXTMETRICW *TM, PFONTGDI FontGDI,
         return;
     }
 
+    ASSERT(pOS2);
+    if (!pOS2)
```

Comment:
Just to know, have we already encountered fonts that do not contain any kind of "OS/2" data structure? What is it for by the way?

Reply:
`MS Sans Serif` is raster font collection (`*.fon`) that doesn't contains the OS/2 table.

Reply:
I had used a static analyzer `cppcheck` to improve the code. This analyzer warned `NULL` check for this code.

Reply:
> `MS Sans Serif` is raster font collection (`*.fon`) that doesn't contain the OS/2 table.

OK but then, would the assert be triggered when using the MS Sans Serif?